theme/templates/resource-landing-page/title-section.html

```diff
@@ -402,20 +402,30 @@ <h2 id="resource-title">{{ title }}</h2>
 	},
 	"license": {
 	    "@type": "CreativeWork",
-	    "text": "{{ cm.metadata.rights.statement }}",
 	    {% if cm.metadata.rights.statement == "This resource is shared under the Creative Commons Attribution CC BY." %}
+	    "text": "{{ cm.metadata.rights.statement }}",
         "url": "http://creativecommons.org/licenses/by/4.0/"
         {% elif cm.metadata.rights.statement == "This resource is shared under the Creative Commons Attribution-ShareAlike CC BY-SA." %}
+        "text": "{{ cm.metadata.rights.statement }}",
         "url": "http://creativecommons.org/licenses/by-sa/4.0/"
         {% elif cm.metadata.rights.statement == "This resource is shared under the Creative Commons Attribution-NoDerivs CC BY-ND." %}
+        "text": "{{ cm.metadata.rights.statement }}",
         "url": "http://creativecommons.org/licenses/by-nd/4.0/"
         {% elif cm.metadata.rights.statement == "This resource is shared under the Creative Commons Attribution-NoCommercial-ShareAlike CC BY-NC-SA." %}
+        "text": "{{ cm.metadata.rights.statement }}",
         "url": "http://creativecommons.org/licenses/by-nc-sa/4.0/"
         {% elif cm.metadata.rights.statement == "This resource is shared under the Creative Commons Attribution-NoCommercial CC BY-NC." %}
+        "text": "{{ cm.metadata.rights.statement }}",
         "url": "http://creativecommons.org/licenses/by-nc/4.0/"
         {% elif cm.metadata.rights.statement == "This resource is shared under the Creative Commons Attribution-NoCommercial-NoDerivs CC BY-NC-ND." %}
+        "text": "{{ cm.metadata.rights.statement }}",
         "url": "http://creativecommons.org/licenses/by-nc-nd/4.0/"
-        {% elif cm.metadata.rights.url %}
+        {% elif cm.metadata.rights.statement and cm.metadata.rights.url %}
+        "text": "{{ cm.metadata.rights.statement }}",
+        "url": "{{ cm.metadata.rights.url }}"
```

Comment:
I would think the whole block could be just line 424 and 425... Why note just 
```
{% if cm.metadata.rights.statement and cm.metadata.rights.url %}
  "text": "{{ cm.metadata.rights.statement }}"
{% endif %}
{% if cm.metadata.rights.url %}
  "url": "{{ cm.metadata.rights.url }}"
{% endif %}
```

Reply:
after talking with Hong, my code snippet won't work because there should only be a comma after text if there is a url